docs/database/couchdb.md

```diff
@@ -14,14 +14,34 @@ described on this page.
 
 ## Attributes
 
-<!-- semconv db.couchdb(full,tag=tech-specific) -->
+<!-- semconv db.couchdb(full) -->
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`db.operation.name`](/docs/attributes-registry/db.md) | string | The HTTP method + the target REST route. [1] | `GET /{db}/{docid}` | `Conditionally Required` [2] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.namespace`](/docs/attributes-registry/db.md) | string | The name of the database, fully qualified within the server address and port. [1] | `customers`; `test.users` | `Conditionally Required` If available. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
did you have a particular mapping in mind for couchdb `db.namespace`? otherwise maybe we could leave it out and only inherit the minimal attributes

Reply:
CouchDB has named databases, e.g. you can create one with [PUT /{db_name}](https://docs.couchdb.org/en/stable/api/database/common.html#put--db) and it participates in every document-related operations.
Do you see any controversy defining `db.namespace` to it? 

Also checking with @maryliag if she has any thoughts.

Reply:
I don't see any problems with using `db.namespace` here. As mentioned by @lmolkova , couchDB have name for the databases that can be used here.

It does have some requirements:
```
The database name {db} must be composed by following next rules:

Name must begin with a lowercase letter (a-z)

Lowercase characters (a-z)

Digits (0-9)

Any of the characters _, $, (, ), +, -, and /.
```

I'm not sure if that is something to be added here, because can be hard to keep track in case it changes, so maybe some pointer to their doc?

Reply:
oh I see the mapping in couchdb's actually matches our generic `db.namespace` definition 🤦‍♂️